client/galaxy/scripts/viz/circster.js

```diff
@@ -634,10 +634,10 @@ var CircsterTrackView = Backbone.View.extend({
             track.get('config').set_value('max_value', self.data_bounds[1], {silent: true});
 
             // Merge chroms layout with data.
-            layout_and_data = _.zip(chrom_arcs, genome_wide_data),
+            var layout_and_data = _.zip(chrom_arcs, genome_wide_data);
 
             // Render each chromosome's data.
-            chroms_data_layout = _.map(layout_and_data, function(chrom_info) {
+            _.each(layout_and_data, function(chrom_info) {
```

Comment:
I may be missing it, but it looks like this is a breaking change.  Instead of mapping onto a new array, this does _.each, but it doesn't actually manipulate the existing array, so no changes are made.